### PR TITLE
Easy to read tiemstamp in log files

### PR DESCRIPTION
**Pull request by Gyeongmin**
Now is the timestamp of the log file is not intuitive.
If you open the log file after a few days it is difficult to make sure that when we work.
And, I will change so that you can specify a timestamp format as an option
Signed-off-by: Gyeongmin Kim <gyeongmintwo@gmail.com>

### CODE REVIEW

File: src/lxc/log.c [hunk 1]
```diff
@@ -82,18 +84,22 @@ static int log_append_stderr(const struct lxc_log_appender *appender,
 static int log_append_logfile(const struct lxc_log_appender *appender,
 			      struct lxc_log_event *event)
 {
+	char date[LXC_LOG_DATEFOMAT_SIZE] = "20150427012246";
```
Comment (stgraber) [inline src/lxc/log.c]: Why that value?

**Comment by stgraber** [comment]
Looks like your branch needs rebasing.

Also can you provide a before/after example?

Thanks!

**Comment by Gyeongmin** [comment]
If you need this patch, I try to rebasing and re-pull

Timestamp format : 2015(Year)04(Month)27(Day)01(Hour)22(Minute)46(Second).277(millisecond)

Now log format:
     lxc-create 1432268972.333 ERROR    lxc_container - lxccontainer.c:create_run_template:1194 - container creation template for a failed
     lxc-create 1432268972.338 ERROR    lxc_create_ui - lxc_create.c:main:274 - Error creating container a

This patch log format:
     lxc-create 20150427012246.277 ERROR    lxc_container - lxccontainer.c:create_run_template:1194 - container creation template for ab failed
     lxc-create 20150427012246.278 ERROR    lxc_create_ui - lxc_create.c:main:274 - Error creating container ab

File: src/lxc/log.c (hunk 1 above)
Comment (Gyeongmin) [inline src/lxc/log.c]: "20150427012246" is Which format is the example being used

**Comment by stgraber** [comment]
Hi,

The branch commits is still a bit of a mess, so I'll squash all of those together for you and push them manually.

Thanks!

**Comment by stgraber** [comment]
Pushed

**Pull request closed without merging**